### ISSUE TITLE
FYI: IETF draft for new UUIDs

### ISSUE THREAD

**Issue by broofa** [WICG/uuid#36]
For those who haven't seen it yet: https://github.com/uuid6/uuid6-ietf-draft

**Comment by marcoscaceres** [comment]
Just wondering, does this new RFC have the potential to impact this draft proposal/API?

**Comment by bcoe** [comment]
> does this new RFC have the potential to impact this draft proposal/API?

IMO, I don't think it does. I believe there were already time based UUIDs, and the main advantage of this new UUID is that it's sortable.

v4 still remains the best choice for purely random UUIDs.

---

Mind you, I think the sortable UUIDs are really interesting (similar to [Twitter's snowflake](https://blog.twitter.com/engineering/en_us/a/2010/announcing-snowflake). Supporting them in the browser would open a can of worms about time based device fingerprinting.

**Comment by bcoe** [comment]
@broofa @ctavan does that agree with your thinking?

**Comment by broofa** [comment]
@bcoe Yup, agreed.  Nothing to worry about as far as this spec goes.  It's focused on time-based IDs, and still in the early stages.

That said, there are a bunch of IDs being thrown around that would be "interesting" if they proved popular and people started asking for them to be back-ported.  E.g. There's talk of 160-bit ids, JSONB and base32 encoding... that sort of thing.
